packit/config/package_config.py

```diff
@@ -137,15 +137,24 @@ def get_from_dict(
             # we want default jobs to go through the proper parsing process
             raw_dict["jobs"] = get_default_jobs()
 
-        if not raw_dict.get("specfile_path", None) and spec_file_path:
-            raw_dict["specfile_path"] = spec_file_path
-
         if not raw_dict.get("upstream_package_name", None) and repo_name:
             raw_dict["upstream_package_name"] = repo_name
 
         if not raw_dict.get("downstream_package_name", None) and repo_name:
             raw_dict["downstream_package_name"] = repo_name
 
+        if not raw_dict.get("specfile_path", None):
+            if spec_file_path:
+                raw_dict["specfile_path"] = spec_file_path
+            else:
+                # we default to downstream_package_name
+                # https://packit.dev/docs/configuration/#specfile_path
+                downstream_package_name = raw_dict.get("downstream_package_name", None)
+                if downstream_package_name:
+                    raw_dict["specfile_path"] = f"{downstream_package_name}.spec"
+                # else: we could try upstream_package_name but this seems
+                # like an extreme corner case
```

Comment:
I can see a similar code in [schema](https://github.com/packit/packit/blob/ecfd705f3c03157a82863fb1b0a710f7695c1037/packit/schema.py#L418) (it is confusing that we do this default values filling on 2 places), so it should be probably already handled by that code if I am not missing anything